tests/test_labnotebook.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function
```

Comment:
we do not need `__future__` imports any longer

Reply:
Removed it.